src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml

```diff
@@ -0,0 +1,37 @@
+id: some_table_id_1
+name: Some Table Name
+hit-policy: UNIQUE
+fields:
+  in:
+    inputOne:
+      description: Some Input 1 Description
+      type: INTEGER
+      expression-type: LITERAL
```

Comment:
this should be redundant

Reply:
intentional, I wanted to have it described in this way